integrations/vello_svg/src/lib.rs

```diff
@@ -33,78 +33,107 @@
 //! - patterns
 
 use std::convert::Infallible;
-
 use usvg::NodeExt;
-use vello::kurbo::{Affine, BezPath, Rect, Stroke};
-use vello::peniko::{Brush, Color, Fill};
-use vello::SceneBuilder;
+use vello::{
+    kurbo::{Affine, BezPath, Point, Rect, Stroke},
+    peniko::{Brush, Color, Fill},
+    SceneBuilder,
+};
+
+/// Re-export vello.
+pub use vello;
 
+/// Re-export usvg.
 pub use usvg;
 
-/// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`], with default error handling
-/// This will draw a red box over (some) unsupported elements
+/// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`], with default error
+/// handling This will draw a red box over (some) unsupported elements
 ///
 /// Calls [`render_tree_with`] with an error handler implementing the above.
 ///
-/// See the [module level documentation](crate#unsupported-features) for a list of some unsupported svg features
-pub fn render_tree(sb: &mut SceneBuilder, svg: &usvg::Tree) {
-    render_tree_with(sb, svg, default_error_handler).unwrap_or_else(|e| match e {});
+/// See the [module level documentation](crate#unsupported-features) for a list
+/// of some unsupported svg features
+pub fn render_tree(sb: &mut SceneBuilder, svg: &usvg::Tree, transform: Option<Affine>) {
+    render_tree_with(sb, svg, transform, default_error_handler).unwrap_or_else(|e| match e {})
 }
 
 /// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`].
 ///
 /// Calls [`render_tree_with`] with [`default_error_handler`].
 /// This will draw a red box over unsupported element types.
 ///
-/// See the [module level documentation](crate#unsupported-features) for a list of some unsupported svg features
+/// See the [module level documentation](crate#unsupported-features) for a list
+/// of some unsupported svg features
 pub fn render_tree_with<F: FnMut(&mut SceneBuilder, &usvg::Node) -> Result<(), E>, E>(
     sb: &mut SceneBuilder,
     svg: &usvg::Tree,
+    transform2: Option<Affine>,
     mut on_err: F,
 ) -> Result<(), E> {
     for elt in svg.root.descendants() {
         let transform = {
-            let usvg::Transform { a, b, c, d, e, f } = elt.abs_transform();
-            Affine::new([a, b, c, d, e, f])
+            let usvg::Transform {
+                sx,
+                kx,
+                ky,
+                sy,
+                tx,
+                ty,
+            } = elt.abs_transform();
+            let (a, b, c, d, e, f) = (
```

Comment:
I'd be tempted to make a single array, then map it through `f64::from` and pass that straight to the constructor

Reply:
Sorry, what? Like this?

```rust
let x: [f64; 6] = [sx, ky, ...].into();
```

Reply:
Yeah, so:
```rust
let arr = [...].map(f64::from);
```

(Where `...` is the components in the right order)